packages/components/src/components/Select/SelectPopover.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { SelectPopover as SelectPopoverPrimitive } from "ariakit/select";
+import type { SelectPopoverProps as SelectPopoverPrimitiveProps } from "ariakit/select";
+import { Box } from "../../primitives/Box";
+
+export interface SelectPopoverProps
+  extends Omit<SelectPopoverPrimitiveProps, "nonce"> {
+  /** The valid contents of the listbox, i.e. SelectItem. */
+  children: NonNullable<React.ReactNode>;
+}
+
+/** A select is an styled dropdown form element that allows users to select a value from a list. */
```

Comment:
What is a SelectPopover?

Reply:
@apvale SelectPopover is the dropdown with the select options.

Reply:
Fixed that. Its different now.